src/components/UI/Buttons/Buttons.tsx

```diff
@@ -4,16 +4,50 @@ import styles from './Buttons.module.sass';
 interface ButtonProps extends JSX.HTMLAttributes<HTMLButtonElement> {
   onClick: () => void;
   children: JSX.Element | string;
+  icon?: string; // Can't put here JSX.Element due to clash with JSX.HTMLAttributes
+  isReversed?: boolean;
+  iconAlt?: string;
 }
 
 /**
  * Button components for the UI
  */
 
```

Comment:
The documentaion should be written to each individual component, and be directly above it (without line breaks).